src/lib/components/collections/req-res-section/sub-components/sub-components-header/Tab.svelte

```diff
@@ -1,28 +1,42 @@
 <script lang="ts">
+  // ---- Icon
   import Crossicon from "$lib/assets/crossicon.svelte";
+  import BookIcon from "$lib/assets/book.svelte";
+  // ----
+
+  // ---- SVG
+  import collectionAsset from "$lib/assets/collection-nodes.svg";
+  import folderTab from "$lib/assets/folder-tab.svg";
+  // ----
+
+  // ---- Enum
   import { ItemType } from "$lib/utils/enums/item-type.enum";
+  // ----
+
+  // ---- helper functions
   import { getMethodStyle } from "$lib/utils/helpers/conversion.helper";
+  // ----
+
+  // ---- Interface
   import type { NewTab } from "$lib/utils/interfaces/request.interface";
-  import collectionAsset from "$lib/assets/collection-nodes.svg";
-  import folderTab from "$lib/assets/folder-tab.svg";
+  // ----
 
-  import BookIcon from "$lib/assets/book.svelte";
   export let tab: NewTab;
-  export let updateCurrentTab: (id: string) => void;
   export let tabWidth: number;
   export let index: number;
-  export let closeTab;
-  export let handleDropOnStart: (index: number) => void;
-  export let handleDropOnEnd: (index: number) => void;
+  export let onTabSelected: (id: string) => void;
```

Comment:
Good work on adding comments for import, you can add comments for these props as well to define a section for events. For each prop, we should also be able to use `/** ... */` JS Docs comments so that it shows up properly when other devs use the component and they would not have to see the actual definition.

Reply:
Done